benchmarks/operator_benchmark/pt/qobserver_test.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import operator_benchmark as op_bench
+import torch
+import torch.quantization.observer as obs
+
+
+qobserver_configs_short = op_bench.config_list(
+    # mode is used to show the direction of the benchmark:
+    # if 'Q', benchmark quantization, else dequantization
+    attr_names=['C', 'M', 'N', 'dtype', 'qscheme'],
+    attrs=[
+        [3, 512, 512, torch.quint8, torch.per_tensor_affine],
+    ],
+    tags=['short']
+)
+
+qobserver_configs_long = op_bench.cross_product_configs(
+    C=[1, 3, 8],
+    M=[256, 1024],
+    N=[256, 1024],
+    dtype=[torch.quint8],  # dtype doesn't change the timing
+    qscheme=[torch.per_tensor_affine, torch.per_tensor_symmetric,
```

Comment:
For observers, cross products wont work.  per-channel qconfig only works with per-channel observer for example. I would write this by having one set of configs for per-tensor-quant (HistogramObserver/MinMax/MovingAverage) and another set of configs for per-channel quant (PerChannelMinMax/PerChannelMovingAverage)
We can fix the dtype to be torch.quint8 for all these tests.